src/KeyboardInputModal.scss

```diff
@@ -0,0 +1,164 @@
+@import '_variables';
+
+.modal {
+    border-radius: 0.25rem;
+}
+
+.modal-body {
+    border-radius: 0.25rem;
+    padding: 0;
+}
+
+.KeyboardInputModal {
+    max-width: 75%;
+}
+
+.KeyboardInputModal__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    row-gap: .5rem;
+}
+
+.KeyboardInputModal__content__title {
+    background-color: #F1F2F4;
+    border-radius: 0.25rem 0.25rem 0 0;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    column-gap: 0.5rem;
+    width: 100%;
+    padding: 0.5rem;
+
+    svg {
+        width: 2rem;
+    }
+}
+
+.KeyboardInputModal__content__toggleBar {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    column-gap: 2rem;
+    margin-left: 3rem;
+}
+
+.KeyboardInputModal__content__toggleBar__label {
+    font-weight: bold;
+}
+
+.KeyboardInputModal__content__toggleBar__toggle {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    column-gap: 0.5rem;
+}
+
+.KeyboardInputModal__content__toggle.ToggleSwitch {
+    .ToggleSwitch__switch-inner-circle {
+        height: 1.9rem;
+        width: 1.9rem;
+    }
+}
+
+.KeyboardInputModal__content__toggle.ToggleSwitch--checked {
+    background-color: #4C9990;
+}
+
+.KeyboardInputModal__content__schemeDropdown {
+    margin-bottom: 0.5rem;
+    margin-left: 3rem;
+    padding: 0.25rem;
+    width: 20rem;
+}
+
+.KeyboardInputModal__content__list {
+    background-color: #F1F2F4;
+    padding: 1rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 1.5rem;
+    row-gap: 1.5rem;
+}
+
+@media only screen and (max-width: 1024px) {
+    .KeyboardInputModal__content__list {
+        display: flex;
+        flex-direction: column;
+        column-gap: 0;
+    }
+}
+
+.KeyboardInputModal__content__list--disabled {
+    display: none;
+}
+
+.KeyboardInputModal__binding {
+    display: grid;
+    grid-template-columns: 6rem auto;
+    column-gap: 1rem;
+    row-gap: 1rem;
+}
+
+.KeyboardInputModal__binding__keyCombo {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    column-gap: 0;
+}
+
+.KeyboardInputModal__binding__icon {
+    border: 2px solid #B9BEC6;
+    background-color: white;
+    border-radius: 3px;
+    box-shadow: 0 3px 1px 0 #B9BEC6;
+    font-weight: bold;
+    height: 2.5rem;
+    padding-top: 0.25rem;
```

Comment:
padding-top pushes the icons to look like vertically centred, but it is not exactly centred. There could be other ways, but I found using flex to center things around convenient. Here is a suggestion, and note that I removed padding-top and text-align.
 
```
.KeyboardInputModal__binding__icon {
display: flex;
justify-content: center;
align-items: center;
border: 2px solid #B9BEC6;    
background-color: white;    
border-radius: 3px;    
box-shadow: 0 3px 1px 0 #B9BEC6;    
font-weight: bold;    
height: 2.5rem;
width: 2.5rem;
}
```

Reply:
Thanks, I had to fudge it and knew there had to be a better way.